src/OpenColorIO/CMakeLists.txt

```diff
@@ -157,7 +157,7 @@ if(NOT WIN32)
     set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
     set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
For consistency, could you change `\${prefix}` to `\${exec_prefix}`?

Reply:
I did that but noticed issues at the same time:
*  To use `CMAKE_INSTALL_LIBDIR` and similars, you need to include `include(GNUInstallDirs)`
*  There is an extra `\` at the start of each variable
*  We only really need `libdir` and don't use `includedir`, my take would be to remove all these and install straight at the relative path, like is done in other places, eg. OCIO apps
```
if(NOT WIN32)
    configure_file(res/OpenColorIO.pc.in ${CMAKE_CURRENT_BINARY_DIR}/OpenColorIO.pc @ONLY)
    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/OpenColorIO.pc DESTINATION lib/pkgconfig)
endif()
```